tools/hangar/__snapshots__/test_corpus/sdk_tests/counter/set.test.w_test_sim.md

```diff
@@ -2,11 +2,10 @@
 
 ## stdout.log
 ```log
-pass ─ set.test.wsim » root/env0/test:set    
-pass ─ set.test.wsim » root/env1/test:key set
+pass ─ set.test.wsim (no tests)
```

Comment:
Regression - tests are not running

Reply:
I found the bug: it's this [suggested change](https://github.com/winglang/wing/pull/4207#discussion_r1332011201). Reverting from `filter` to `testFilter` fixed it even though I'm not sure why.